protected/tests/functional_test/OnlyLoggedInTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
I would have added this file into the `protected/tests/functional` directory.

Maybe think about renaming this file to something more informative, maybe `AdminSiteTest.php` or `AdminSiteAccessTest.php`?

Also, look at the names of the functions in the other functional test files. The function names have a pattern: `testItShould...`. Its probably best to keep with this pattern for consistency.